src/pydata_sphinx_theme/assets/styles/abstracts/_color.scss

```diff
@@ -70,18 +76,18 @@
 
   // Loop from the second character (omitting #)
   @for $i from 2 through $length {
-    $c: str-slice($string-lower, $i, $i);
+    $c: string.slice($string-lower, $i, $i);
 
     // If wrong character, return
-    @if index($hex, $c) == null {
+    @if not list.index($hex, $c) {
```

Comment:
There were a lot of things that caused the stylelint hook to fail. This was one of them, but the pre-commit hook would not autofix this line. There were a lot of lines that it did not autofix but that seemed like they should be autofixable. I'm not familiar enough with SCSS to know if `@if index($hex, $c) == null` is equivalent to `@if not list.index($hex, $c)` so this change makes me just a tiny bit wary.

Reply:
Thanks for clarifying!